src/branding-bar/BrandingBar.js

```diff
@@ -1,8 +1,62 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
-const BrandingBarContainer = styled.header``;
+import LogoIconSrc from "../assets/icons/recidiviz_logo.svg";
+import { LinkPill } from "../pill";
+
+const brandingBarFlexProperties = css`
+  align-items: center;
+  display: flex;
+`;
+
+const BrandingBarContainer = styled.header`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 50px;
+`;
+
+const BrandingBarHeader = styled.div`
+  ${brandingBarFlexProperties}
+`;
+
+const Logo = styled.img``;
+
+const BrandingBarTitle = styled.h1`
+  color: ${(props) => props.theme.colors.heading};
+  font: ${(props) => props.theme.fonts.display};
+  display: inline;
+  margin-left: 16px;
+`;
+
+const BrandingBarLinkContainer = styled.div``;
+
+const BrandingBarLinksList = styled.ul`
+  ${brandingBarFlexProperties}
+  justify-content: space-between;
+`;
+
+const BrandingBarLink = styled.li`
+  list-style-type: none;
+  margin-left: 16px;
+`;
 
 export default function BrandingBar() {
-  return <BrandingBarContainer>Branding bar</BrandingBarContainer>;
+  return (
+    <BrandingBarContainer>
+      <BrandingBarHeader>
+        <Logo alt="Recidiviz" src={LogoIconSrc} />
+        <BrandingBarTitle>North Dakota Corrections</BrandingBarTitle>
```

Comment:
I've hard-coded "North Dakota Corrections" for now, but would assume we'd like to use some sort of `TenantContext` in the future?

Reply:
In the future, absolutely. For now, we want to basically hard-code ND values but not make major design choices that might box us into a single-tenant approach without major effort.